app/inbound/save-settlement.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../data')
+
+const saveSettlement = async (settlement, transaction) => {
+  await db.settlement.create({ ...settlement }, { transaction })
```

Comment:
can be just `await db.settlement.create(settlement, { transaction })`

(I think, been meaning to try it out)